builder/draft-claise-yang-full-include.xml

```diff
@@ -64,6 +64,8 @@
             In that RFC, three different phases of data model life cycle are identified: "design time", "implementation time" and "run time".
             Only the last two are covered.
             We focus here on the first phase of the life cycle, that is inserting modules at design time.
+            Our aim is to propose a feature similar to those supported in other data-modelling languages,
```

Comment:
I would move that to a single paragraph between the one that speaks about schema mount and peer mount and the one that starts with "in this document".
we could also add the reference to RELAX-NG from Lada